### ISSUE TITLE
Why does optimus-manager blacklist nvidia modules in /usr/lib/modprobe.d/optimus-manager.conf ?

### ISSUE THREAD

**Issue by sukulent** [Askannz/optimus-manager#507]
Inside the file there is:
```
blacklist nouveau
blacklist nvidia_drm
blacklist nvidia_uvm
blacklist nvidia_modeset
blacklist nvidia
```
I need nvidia_uvm module to be loaded at boot, I can load it manually using `nvidia-modprobe -u` after, but that should be automatic. In logs it says `systemd-modules-load[252]: Module 'nvidia_uvm' is deny-listed`, but it loads the other modules fine.

So my question is, what am I missing and how should I make nvidia_udm load at boot?
This is a bit confusing.

I am running Arch (5.18.14-zen1-1-zen) on Lenovo Legion 5, RTX 2060 and i5-10300H

**Comment by afro-coder** [comment]
Is that file created by optimus manager? If not you can simply delete the line

To check if that file is owned by a package use
```
pacman -Qo /etc/modprobe.d/nvidia.conf
```
Also which mode is optimus manager running in?
```
optimus-manager --status
```

Depending on your mode you can also automate it using the files under `/etc/optimus-manager/xsetup-*`

**Comment by lu0se** [comment]
file is owned by the package use
`/usr/lib/modprobe.d/optimus-manager.conf is owned by optimus-manager 1.4-5
`
and I mentioned that in [ Askannz/optimus-manager/package/PKGBUILD  ](https://github.com/Askannz/optimus-manager/blob/master/package/PKGBUILD)
`54 install -Dm644 modules/optimus-manager.conf "$pkgdir/usr/lib/modprobe.d/optimus-manager.conf"`

**Comment by brainos233** [comment]
It can cause `nvidia nvidia_drm nvidia_uvm nvidia_modeset` can't be auto loaded when fully disable integrated graphics in UEFI settings,

Must modprobe them manaully after startup

**Comment by nwildner** [comment]
Closing case since this question was answered back in 2022 and the original reported didn't came back. 

Please fell free to reopen it if there are still any pending queries.

**Comment by sukulent** [comment]
Oh. Hello there, I am sorry, I have no idea now why I made this issue. I had started Uni after I made it work somehow and forgotten about this.
I have just read the explainer issue about the state of this project and now I get why the package name change surprised when updating. Sadly, I had to revert to older version as I did not have the time to debug - it did not work. I will try to do something about this, as nvidia update or something broke gpu going into D3cold on hybrid and I haven't updated in a while.
I will create new issue, if relevant.
Thanks for helping now with this project, I had tried the other stuff but this worked for 2 years quite well, and I beat friends identical laptop w/windows battery life lol.
